rand_chacha/README.md

```diff
@@ -0,0 +1,47 @@
+# rand_chacha
+
+[![Build Status](https://travis-ci.org/rust-random/rand.svg)](https://travis-ci.org/rust-random/rand)
+[![Build Status](https://ci.appveyor.com/api/projects/status/github/rust-random/rand?svg=true)](https://ci.appveyor.com/project/dhardy/rand)
```

Comment:
I just changed this to: https://ci.appveyor.com/project/rust-random/rand

Reply:
I'll change it for all crates.